ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -169,25 +169,56 @@ fi
 PRIVATE_LINK_SWITCH=""
 if [[ "$PRIVATE_LINK" == "true" ]]; then
   PRIVATE_LINK_SWITCH="--private-link"
-
-  PRIVATE_SUBNET_IDs=$(cat ${SHARED_DIR}/private_subnet_ids | tr -d "[']")
-  if [[ -z "${PRIVATE_SUBNET_IDs}" ]]; then
-    echo -e "The privated_subnet_ids are mandatory."
-    exit 1
-  fi
-  SUBNET_ID_SWITCH="--subnet-ids ${PRIVATE_SUBNET_IDs}"
+  ENABLE_BYOVPC="true"
+  PRIVATE_SUBNET_ONLY="true"
```

Comment:
We need to declare ` ENABLE_BYOVPC` and `PRIVATE_SUBNET_ONLY` out of `if` condition to avoid the error `unbound variable`.